package-scripts/omnibus-toolchain/postrm

```diff
@@ -8,9 +8,12 @@
 # - if you are under 40, get peer review from your elders.
 #
 
-# Clean up the install directory if they exists
-if [ -d /opt/omnibus-toolchain ]; then
-  rm -rf /opt/omnibus-toolchain
+if [ ! -f /etc/redhat-release -a ! -f /etc/fedora-release -a ! -f /etc/system-release -a ! -f /etc/SuSE-release ]; then
+  # no op, will clean itself up
```

Comment:
This is not going to noop as you would expect, instead it will either hang indefinately or error. 

Noop in bash is `:`.